package-lock.json

```diff
@@ -1,9342 +1,8 @@
 {
   "name": "national-npp",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```

Comment:
These changes to package-lock.json are the result of installing chakra's icons using npm install.  We want to avoid using npm install so we do not run into situations like this.  To address this issue, remove this package-lock.json file and replace it with the one on the main branch.  Run npm ci and you will then have the correct set up.  The difference between npm ci and npm install that we care about is that npm ci will not change this file.  Always use npm ci.